### PR TITLE
SEO page metadata

### PR DESCRIPTION
**Pull request by tan-90**
Added metadata to all pages.
Changed pages to pass metadata to the layout component.
Fixed wrong image path on long-form content.

### CODE REVIEW

File: src/hooks/useSiteMetadata.tsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ const defaultMeta = {
   title: 'Default title',
   description: 'Default Description',
   url: 'default',
+  siteUrl: 'default',
```
Comment (Lisenish) [inline src/hooks/useSiteMetadata.tsx:7]: Strange, why it was deleted?

Comment (Lisenish) [reply src/hooks/useSiteMetadata.tsx:7]: BTW, should we refactor this at last? (url & siteUrl duplicates)

File: src/pages/demo/index.tsx [hunk 2]
```diff
@@ -1,26 +1,34 @@
...
+  pageContext: GeneratedPageContext
+}
+export default function DemoPage({ pageContext }: DemoPageProps) {
   const breadcrumbs = pageContext.breadcrumb.crumbs
+  const { pathname } = useLocation()
+  let siteMetadata = useSiteMetadata()
+  siteMetadata = {
+    ...siteMetadata,
+    title: 'Technical Demo - FingerprintJS Pro',
+    url: `${siteMetadata.url}${pathname}`,
```
Comment (Lisenish) [inline src/pages/demo/index.tsx:22]: Let's use siteUrl instead and replace all places to use it.

File: src/pages/pricing/index.tsx [hunk 3]
```diff
@@ -1,6 +1,5 @@
 import React from 'react'
-import Helmet from 'react-helmet'
-import Layout from '../../components/Layout'
+import { LayoutTemplate } from '../../components/Layout'
```
Comment (Lisenish) [inline src/pages/pricing/index.tsx:2]: I think maybe we should make regular Layout named export as well? So stimulate conscious choice between the two.

Comment (tan-90) [reply src/pages/pricing/index.tsx:2]: Agreed!

**Review by Lisenish — APPROVED** [review]

**Review by Lisenish — APPROVED** [review]

**Pull request merged**